### PR TITLE
Add Recipe for Darwin Font

### PR DESCRIPTION
**Pull request by jaypmorgan**
"Darwin is a free and open-source ty­pe­fa­ce project aiming to develop a com­pre­hen­sive set of fonts intended for book and article writing, with a particular focus on aca­de­mic and scientific usage."

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [ ] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [ ] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=pullrequests).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/font/font-d/font-darwin.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# typed: true
+# frozen_string_literal: true
+
+cask "font-darwin" do
```
Comment (Aaron-212) [inline Casks/font/font-d/font-darwin.rb:4]:
```suggestion
cask "font-darwin" do
```
There should be no comments in Cask declaration.

File: Casks/font/font-d/font-darwin.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/topological-modular-forms/Darwin-Typeface/raw/main/output/DarwinSerif-Regular.ttf"
```
Comment (Aaron-212) [inline Casks/font/font-d/font-darwin.rb:8]:
```suggestion
  url "https://github.com/topological-modular-forms/Darwin-Typeface/raw/main/output/DarwinSerif-Regular.otf"
```
OTF is preferred over TTF

File: Casks/font/font-d/font-darwin.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  name "Darwin"
+  homepage "https://github.com/topological-modular-forms/Darwin-Typeface/tree/main"
```
Comment (Aaron-212) [inline Casks/font/font-d/font-darwin.rb:10]:
```suggestion
  homepage "https://github.com/topological-modular-forms/Darwin-Typeface"
```

File: Casks/font/font-d/font-darwin.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+  font "DarwinSerif-Regular.ttf"
```
Comment (Aaron-212) [inline Casks/font/font-d/font-darwin.rb:12]:
```suggestion
  font "DarwinSerif-Regular.otf"
```

**Review by Aaron-212 — COMMENTED** [review]
Suggest some changes

**Review by Aaron-212 — COMMENTED** [review]
Don't request me to review😂 I don't have access to approve PR

**Comment by Aaron-212** [comment]
@miccal Please review

**Comment by jaypmorgan** [comment]
Sorry, I thought you had the privilege! :-D

File: Casks/font/font-d/font-darwin.rb [hunk 5]
```diff
@@ -0,0 +1,12 @@
+cask "font-darwin" do
+  version :latest
+  sha256 :no_check
+
+  url "https://github.com/topological-modular-forms/Darwin-Typeface/raw/main/output/DarwinSerif-Regular.otf"
+  name "Darwin"
```
Comment (miccal) [inline Casks/font/font-d/font-darwin.rb:6]:
```suggestion
  name "Darwin"
  desc "Typeface for books and articles with a focus on scientific writing"
```

**Review by miccal — APPROVED** [review]

**Review by krehel — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @jaypmorgan.

**Pull request merged**